ci/travis/format.sh

```diff
@@ -38,8 +38,11 @@ fi
 if which clang-format >/dev/null; then
   CLANG_FORMAT_VERSION=$(clang-format --version | awk '{print $3}')
   if [[ $CLANG_FORMAT_VERSION != "7.0.0" ]]; then
-    echo "Please install clang-format 7.0.0. You currently are using $CLANG_FORMAT_VERSION."
-    exit 1
+    echo "WARNING: Please install clang-format 7.0.0. You currently are using $CLANG_FORMAT_VERSION."
+    read -p "Are you sure to continue?[y/n]" input
+    if ! [ $input = 'y' ]; then
+        exit 1
+    fi
```

Comment:
@robertnishihara  Can I add a continue check here? As I know, the popular one on Mac is clang-format 8.0 at present. Format code use 8.0 by this script and change a little difference is really convenient.

Reply:
I think a warning is enough. 
Most of time, clang-format 7.0 and 8.0 generates the same result.
We just need to let users be aware that the results could be different. 

Also, it seems not very easy to install clang-format 7.0 on macOS. [This script](https://gist.github.com/ffeu/0460bb1349fa7e4ab4c459a6192cbb25#gistcomment-2765744) can work. But I guess not everyone would like to spend time building clang-format from source.

But, can you change this in a separate PR?